models/money/src/builder.spec.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
```

Comment:
Same question

Reply:
Addressed in 588274198e7103b275307fe755d79761bad755e3